eng/pipelines/common/xplat-setup.yml

```diff
@@ -97,6 +97,16 @@ jobs:
           name: NetCoreInternal-Pool
           queue: BuildPool.Ubuntu.1604.Amd64
 
+        # Public FreeBSD Build Pool
+        ${{ if and(eq(parameters.osGroup, 'FreeBSD_x64'), eq(variables['System.TeamProject'], 'public')) }}:
```

Comment:
I don’t think this is going to differ ever from the Linux pools as this is now contenirized. Could we merge the two conditions instead?

Reply:
Sure.  I wasn't clear on the future of how this works as we might want to have build agents running natively on BSD when it becomes possible.